components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/ExtensionsTest.kt

```diff
@@ -30,4 +31,18 @@ class ExtensionsTest {
 
         assertEquals("Hello", map.translate())
     }
+
```

Comment:
Let's add a test for Addon.translatedName as well as it's easy it shouldn't remain untested. :)